config.json

```diff
@@ -356,56 +356,56 @@
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-mlkit-barcode-scanning",
-        "version": "18.0.0",
-        "nugetVersion": "118.0.0.2",
+        "version": "18.1.0",
+        "nugetVersion": "118.1.0",
         "nugetId": "Xamarin.GooglePlayServices.MLKit.BarcodeScanning",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-mlkit-face-detection",
-        "version": "17.0.1",
-        "nugetVersion": "117.0.1.2",
+        "version": "17.1.0",
+        "nugetVersion": "117.1.0",
         "nugetId": "Xamarin.GooglePlayServices.MLKit.FaceDetection",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-mlkit-image-labeling",
-        "version": "16.0.7",
-        "nugetVersion": "116.0.7.1",
+        "version": "16.0.8",
+        "nugetVersion": "116.0.8",
         "nugetId": "Xamarin.GooglePlayServices.MLKit.ImageLabeling",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-mlkit-language-id",
-        "version": "17.0.0-beta1",
-        "nugetVersion": "117.0.0.2-beta1",
+        "version": "17.0.0",
+        "nugetVersion": "117.0.0",
```

Comment:
This needs to be `117.0.0.2`, everything else looks good.

Reply:
hectic trying to catch up.